src/webview/contextMenuBuilder.ts

```diff
@@ -261,7 +347,15 @@ export class ContextMenuBuilder {
   ): Electron.CrossProcessExports.Menu {
     const menu = new Menu();
 
+    // @ts-expect-error Property 'enableTranslator' does not exist on type 'ContextMenuParams'.
```

Comment:
Can we extend the ContextMenuParams type and add our own properties? So we don't need to use these ts-expect-errors

Reply:
I agree... But I've never done so. Can you give me an example?

I'm trying to do this:
```
interface IProps extends Electron.ContextMenuParams {
  info: any;
  menuInfo: any;
}

export default class FerdiumContextMenuParams implements IProps {
  x: number;

  y: number;

  frame: Electron.WebFrameMain;

  linkURL: string;

  linkText: string;

  pageURL: string;

  frameURL: string;

  srcURL: string;

  mediaType:
    | 'none'
    | 'image'
    | 'audio'
    | 'video'
    | 'canvas'
    | 'file'
    | 'plugin';

  hasImageContents: boolean;

  isEditable: boolean;

  selectionText: string;

  titleText: string;

  altText: string;

  suggestedFilename: string;

  selectionRect: Electron.Rectangle;

  selectionStartOffset: number;

  referrerPolicy: Electron.Referrer;

  misspelledWord: string;

  dictionarySuggestions: string[];

  frameCharset: string;

  inputFieldType: string;

  spellcheckEnabled: boolean;

  menuSourceType:
    | 'none'
    | 'mouse'
    | 'keyboard'
    | 'touch'
    | 'touchMenu'
    | 'longPress'
    | 'longTap'
    | 'touchHandle'
    | 'stylus'
    | 'adjustSelection'
    | 'adjustSelectionReset';

  mediaFlags: Electron.MediaFlags;

  editFlags: Electron.EditFlags;

  info: any;

  menuInfo: any;
}
```

But it is throwing me a bunch of errors:
![image](https://user-images.githubusercontent.com/37463445/184204696-7d74b2f1-1cb5-47d9-b675-22dc1d01b4e8.png)